src/ResourceManager/Compute/Commands.Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -36,7 +36,8 @@ public class NewAzureRmDiskConfigCommand : Microsoft.Azure.Commands.ResourceMana
             Mandatory = false,
             Position = 0,
             ValueFromPipelineByPropertyName = true)]
-        public StorageAccountTypes? AccountType { get; set; }
+        [Alias("AccountType")]
```

Comment:
Should we deprecate the usage of this alias?  We can write a warning if the user used the alias using MyInvocation.Line, or some other properties.  This is not a required change, it can be done later